src/Adapter/Product/Combination/Repository/CombinationRepository.php

```diff
@@ -366,4 +368,24 @@ public function getCombinationIdsByAttributes(ProductId $productId, array $attri
             return new CombinationId((int) $combination['id_product_attribute']);
         }, $result);
     }
+
+    public function updateCombinationStock(ProductId $productId, OutOfStockType $outOfStockType, ShopConstraint $shopConstraint = null): void
+    {
+        $qb = $this->connection->createQueryBuilder();
+        $qb
+            ->update(sprintf('%sstock_available', $this->dbPrefix), 'ps')
+            ->set('ps.out_of_stock', (string) $outOfStockType->getValue())
```

Comment:
```suggestion
            ->set('ps.out_of_stock', (string) $outOfStockType->getValue())
```

DB prefix is dynamic :D